docs/examples/en/lines/LineMaterial.html

```diff
@@ -64,7 +64,7 @@ <h3>[property:number gapSize]</h3>
 		<p>The size of the gap. Default is `1`.</p>
 
 		<h3>[property:Float linewidth]</h3>
-		<p>Controls line thickness. Default is `1`.</p>
+		<p>Controls line thickness in screen pixels when [page:worldUnits] is `false` (default), or in world units when [page:worldUnits] is `true`. Default is `1`. Values less than 1 pixel may not be visible when using screen space units.</p>
```

Comment:
>in screen pixels

in CSS pixel units

>Values less than 1 pixel may not be visible when using screen space units

I would remove this comment, as it is not definitive. We do not explicitly define the behavior when the width is fractional.